frontend/src/app/oe/components/block-rate-strike-details-v2/block-rate-strike-details-v2.component.ts

```diff
@@ -80,8 +79,8 @@ export class BlockRateStrikeDetailsV2Component implements OnInit {
               (strikeHeight - this.latestBlock.height) * 600;
           }
 
-          if (fromBlockHeight > strikeHeight) {
-            fromBlockHeight = strikeHeight - 13;
+          if (!fromBlockHeight) {
+            fromBlockHeight = strikeHeight - 14;
```

Comment:
looks like we need
```
fromBlockHeight = max (0, strikeHeight - 14);
```
here in order to not to work with negative fromBlockHeight?

Reply:
Done